src/main/resources/logback.xml

```diff
@@ -87,7 +87,7 @@
 
     </Logger>
 
-    <Logger name="beam.router.BeamRouter" level="ERROR" additivity="false">
+    <Logger name="beam.router.BeamRouter" level="INFO" additivity="false">
```

Comment:
Please make sure to convert this back to ERROR before merging

Reply:
ah. good catch.

Reply:
ah, good catch.